### ISSUE TITLE
Missing AppIcon-Transparent.png

### ISSUE THREAD

**Issue by alexchantastic** [WilliamLCobb/iNDS#2]
Build fails due to missing AppIcon-Transparent.png

```
CopyPNGFile /Users/foo/Library/Developer/Xcode/DerivedData/iNDS-gtvujqbcpjjrlngkpcbcvoeljzuz/Build/Products/Debug-iphoneos/iNDS.app/AppIcon-Transparent.png Resources/required/AppIcon-Transparent.png
    cd /Users/foo/Downloads/iNDS-master
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/usr/bin/copypng -compress /Users/foo/Downloads/iNDS-master/Resources/required/AppIcon-Transparent.png /Users/foo/Library/Developer/Xcode/DerivedData/iNDS-gtvujqbcpjjrlngkpcbcvoeljzuz/Build/Products/Debug-iphoneos/iNDS.app/AppIcon-Transparent.png

ERROR: Can't find /Users/foo/Downloads/iNDS-master/Resources/required/AppIcon-Transparent.png
Command /Applications/Xcode.app/Contents/Developer/usr/bin/copypng failed with exit code 1
```

**Comment by Zenivo** [comment]
I'm having the same issue. It makes me unable to build :/

**Comment by pmp174** [comment]
If the issue has been fixed it should be closed.

**Comment by DylanTackoor** [comment]
Also had the same problem, does indeed seem to fixed now though.

**Comment by alexchantastic** [comment]
Yep seems like this has been fixed with cd00483089b203dd13c16d0978045ed7d3f4d665. Closing :+1:!
